dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
PrivateTmp=true is a recommended secure option by Systemd team.
Since it doesn't interfere with work, it can be left as is.

But as you request, I'll remove it.